### DISCUSSION TITLE
Why is storedSessions deprecated?!

### DISCUSSION THREAD

**Discussion by remihuigen** [keystonejs/keystone#9212] [Questions] [answered]
Why is [storedSessions()](https://github.com/keystonejs/keystone/blob/fadde1fad45cb7e698f421a705d0bf5942216105/packages/core/src/session/index.ts#L128) being deprecated? I want to use a Redis store for my sessions, as per [this example](https://github.com/keystonejs/keystone/tree/fadde1fad45cb7e698f421a705d0bf5942216105/examples/custom-session-redis)

**Comment by dcousens** [comment, marked as answer]
@remihuigen I think we should remove the `@deprecated` tag, to be honest.
@gautamsi surfaced this again in https://github.com/keystonejs/keystone/pull/9191, and I honestly don't have a good answer now.

This was a hot topic when working through https://github.com/keystonejs/keystone/pull/8097 internally, and a number of different pull requests, but I think we were too eager in tagging this as `@deprecated`.

The abstraction provided by `storedSession` is useful, despite being easy to replicate, and I don't think it should be removed.
